### PR TITLE
Bug in reactive power dispatching 

### PR DESCRIPTION
**Pull request by annetill**
**Please check if the PR fulfills these requirements**

- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**

No.

**What kind of change does this PR introduce?**

Double.NaN as Q in the terminal of a generator that has been discarded from voltage control.

**What is the current behavior?**

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?**
- [ ] Yes
- [ ] No

**If yes, please check if the following requirements are fulfilled**

- [ ] The *Breaking Change* or *Deprecated* label has been added
- [ ] The migration steps are described in the following section

**What changes might users need to make in their application due to this PR? (migration steps)**

**Other information**:

### CODE REVIEW

File: src/test/java/com/powsybl/openloadflow/ac/AcLoadFlowGeneratorTest.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+    @BeforeEach
+    void setUp() {
+        loadFlowRunner = new LoadFlow.Runner(new OpenLoadFlowProvider(new DenseMatrixFactory()));
+        parameters = new LoadFlowParameters();
+        parametersExt = OpenLoadFlowParameters.create(parameters);
+    }
+
+    @Test
+    void testWithCondenser() {
```
Comment (annetill) [inline src/test/java/com/powsybl/openloadflow/ac/AcLoadFlowGeneratorTest.java:40]: This test is just to show that we can support voltage control of synchronous machine with only reactive capability.

**Review by vidaldid-rte — APPROVED** [review]
OK for me.

File: src/main/java/com/powsybl/openloadflow/network/impl/LfDanglingLineGenerator.java [hunk 2]
```diff
@@ -71,7 +71,7 @@ public OptionalDouble getRemoteControlReactiveKey() {
 
     @Override
     public double getTargetQ() {
-        return getDanglingLine().getGeneration().getTargetQ() / PerUnit.SB;
+        return Networks.fixValue(getDanglingLine().getGeneration().getTargetQ()) / PerUnit.SB;
```
Comment (rosiereflo) [inline src/main/java/com/powsybl/openloadflow/network/impl/LfDanglingLineGenerator.java:74]: I find the name fixValue unclear. Maybe redefineIfNan, or something similar?

Comment (annetill) [reply src/main/java/com/powsybl/openloadflow/network/impl/LfDanglingLineGenerator.java]: Done

Comment (rosiereflo) [reply src/main/java/com/powsybl/openloadflow/network/impl/LfDanglingLineGenerator.java]: thanks!

**Review by rosiereflo — APPROVED** [review]

**Pull request merged**